fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/ResourceManagerHelperImpl.java

```diff
@@ -173,4 +193,9 @@ public void registerReloadListener(IdentifiableResourceReloadListener listener)
 			throw new RuntimeException("Listener with previously unknown ID " + listener.getFabricId() + " already in listener set!");
 		}
 	}
+
+	private static ResourcePackSource getResourceSource(String modId) {
+		String trimmedId = StringUtils.abbreviate(modId, 8);
```

Comment:
Not sure trimming the modid is a good idea.

Reply:
ends up going over to the hidden second line if the modid is too long